### PR TITLE
Required refactor before upgrading Tagger

### PR DESCRIPTION
**Pull request by alainbodiguel**

### CODE REVIEW

File: arlas-core/src/main/java/io/arlas/server/services/ExploreService.java [hunk 1]
```diff
@@ -44,28 +44,24 @@
...
     protected CollectionReferenceDao daoCollectionReference;
     protected ResponseCacheManager responseCacheManager;
 
     public ExploreService() {
     }
 
-    public ExploreService(ArlasServerConfiguration configuration) {
-        this.configuration = configuration;
-        this.responseCacheManager = new ResponseCacheManager(configuration.arlasrestcachetimeout);
+    public ExploreService(String baseUri, int arlasrestcachetimeout) {
```
Comment (laurent-thiebaud-gisaia) [inline arlas-core/src/main/java/io/arlas/server/services/ExploreService.java:54]: CamelCase ;)

Comment (alainbodiguel) [reply arlas-core/src/main/java/io/arlas/server/services/ExploreService.java]: Fixed

File: arlas-server/src/main/java/io/arlas/server/impl/ElasticDatabaseToolsFactory.java [hunk 2]
```diff
@@ -41,16 +41,20 @@
...
+    private WFSToolService wfsService;
 
     public ElasticDatabaseToolsFactory(ArlasServerConfiguration configuration) {
         super(configuration);
         this.elasticClient = new ElasticClient(configuration.elasticConfiguration);
-        this.exploreService = new ElasticExploreService(elasticClient, configuration);
         this.collectionReferenceDao = new ElasticCollectionReferenceDao(elasticClient, configuration.arlasindex, configuration.arlascachesize, configuration.arlascachetimeout);
-        this.ogcDao = new ElasticOGCCollectionReferenceDao(elasticClient, collectionReferenceDao, configuration.arlasindex, Service.CSW);
-        this.wfsService = new ElasticWFSToolService((ElasticExploreService) exploreService);
+        this.exploreService = new ElasticExploreService(elasticClient, collectionReferenceDao, configuration.arlasBaseUri, configuration.arlasrestcachetimeout);
```
Comment (laurent-thiebaud-gisaia) [inline arlas-server/src/main/java/io/arlas/server/impl/ElasticDatabaseToolsFactory.java:51]: also this one isn't camel cased

Comment (alainbodiguel) [reply arlas-server/src/main/java/io/arlas/server/impl/ElasticDatabaseToolsFactory.java:51]: This one comes from the configuration class which I haven't touched. I could fix it though, as there are many variables in it which are not camel case.

Comment (alainbodiguel) [reply arlas-server/src/main/java/io/arlas/server/impl/ElasticDatabaseToolsFactory.java]: Done

**Pull request merged**